engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -184,6 +186,12 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
       }
     // Expression
     case Event(ExpressionEvaluationSucceededResponse(expressionKey, callOutputs), stateData) =>
+      expressionKey.node match {
+        case _: ExposedExpressionNode | _: ExpressionBasedGraphOutputNode =>
+          workflowLogger.info(s"Expression evaluation succeeded: '${expressionKey.node.fullyQualifiedName}' (scatter index: ${expressionKey.index}, attempt: ${expressionKey.attempt})")
```

Comment:
Do we really want this? Discuss...

Reply:
Given the potentially enormous amount of expressions a workflow can have I'd be reluctant to log this at `info` level, also it's unclear how much value it provides to the user ?
Maybe at a debug level ?

Reply:
What do you think about the subworkflow and job succeeded equivalents? 

FWIW I only added these for sanity-checking and also because cache hit successes weren't being reported at all

Reply:
Yeah I think they make sense, although I though we already logged that but maybe not in the workflow log per se